pkg/cosign/image_signer.go

```diff
@@ -111,6 +134,139 @@ func (s *ImageSigner) sign(ro *options.RootOptions, refImage, secretLocation, co
 	return nil
 }
 
+func (s *ImageSigner) SignBuilder(
+	ctx context.Context,
+	imageReference string,
+	cosignSecrets []corev1.Secret,
+	builderKeychain authn.Keychain,
+) ([]v1alpha2.SignaturePath, error) {
+	const Timeout time.Duration = time.Second * 30
+
+	signaturePaths := make([]v1alpha2.SignaturePath, 0, 10)
```

Comment:
Any particular reason to use an initial capacity of 10?

Reply:
No specific reason, will remove this too.

Reply:
Removed.